Bottom: fac77e11574baf1034f2881bef2086f2f30ac95a
Top:    1944ff9455330a5668da3d69575cd2e1928f07dc
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-19 15:13:57 -0700

Refresh of rstream-fork

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 8595657..490caa5 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -557,7 +557,9 @@ static int run(void)
 			init_latency_test(test_size[i].size);
 			run_test();
 		}
-		if (!fork_pid)
+		if (fork_pid)
+			wait(NULL);
+		else
 			rs_shutdown(rs, SHUT_RDWR);
 		rs_close(rs);
 
@@ -583,7 +585,9 @@ static int run(void)
 			ret = run_test();
 	}
 
-	if (!fork_pid)
+	if (fork_pid)
+		wait(NULL);
+	else
 		rs_shutdown(rs, SHUT_RDWR);
 	rs_close(rs);
 free:
